### PR TITLE
Creates code structure to make app responsive

### LINKED ISSUE #15: Make it Responsive

### PR DESCRIPTION
**Pull request by italomarca**
We wan't to make this app responsive (#15), in order to do that, I divided this task in some parts:
1. Create code structure with symlinks so that we are able to reuse some web components and only rewrite what we need
2. Rewrite components so that they work well on web and mobile

This PR takes care of part 1 by:
- creating a web/mobile layer (most of the mobile layer is just symlinked from the web layer for now)
- and using [react-responsive](https://www.npmjs.com/package/react-responsive) to know if app is being used on a desktop or a phone

Helpfull links:
- https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/

### CODE REVIEW

File: src/mobile/pages/home.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  const updateMovie = async () => {
+    const movie = await moviedbService.getMovie()
+    setMovie(movie)
+  }
+  
+  return (
+    <>
+      <Header />
+
```
Comment (italomarca) [inline src/mobile/pages/home.js:25]: Aqui só tem o header pq ainda não fiz a home mobile.

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
-import App from './App';
-// import reportWebVitals from './reportWebVitals';
+import HomeWeb from './web/pages/home'
+import HomeMobile from './mobile/pages/home'
+
+import { useMediaQuery } from 'react-responsive'
+
+const App = () => {
+  const isDesktopOrLaptop = useMediaQuery({
+    query: '(min-device-width: 1224px)'
```
Comment (scamardi) [inline src/index.js:11]: isso aqui pega o tamanho da tela do cara para saber se é mobile? é isso? 🤔

File: src/mobile/components/Header.js [hunk 3]
```diff
@@ -0,0 +1 @@
+../../web/components/Header.js
```
Comment (scamardi) [inline src/mobile/components/Header.js:1]: vc criou pastas para linkar o carregamento do front para mobile?

File: src/mobile/pages/home.js (hunk 1 above)
Comment (scamardi) [reply src/mobile/pages/home.js:25]: esse é o header atual?

**Review by scamardi — APPROVED** [review]
🤯

**Pull request merged**